signac/core/synced_collection.py

```diff
@@ -82,6 +91,13 @@ def from_base(cls, data, backend=None, **kwargs):
                 return data.item()
         return data
 
+    @classmethod
+    def from_backend(cls, backend):
+        for _cls in cls.backend_registry:
+            if _cls.backend == backend:
+                return _cls
+        raise ValueError("{backend} backend not found.".format(backend=backend))
```

Comment:
We can use f-strings here:
```suggestion
        raise ValueError(f"{backend} backend not found.")
```